third_party/amd/lib/TritonAMDGPUTransforms/RescheduleOps.cpp

```diff
@@ -119,6 +182,8 @@ struct Graph {
     return copy;
   }
 
+  void setNotesPriorities(PriorityStrategy &&strategy) { strategy.set(nodes); }
```

Comment:
should this be setNodesPriorities ?

Reply:
You are correct. I renamed `NodesPriorities` to `NodesWeights`